### ISSUE TITLE
Use "external player" as playback feature?

### ISSUE THREAD

**Issue by lolento** [jellyfin/jellyfin-android#67]
Can we have this feature added in?  I prefer to use Mx Player as it has more aspect ratio selections.

**Comment by lolento** [comment]
Yes, the selection is an option under the user profile's playback option.
![screenshot_20190201-090836_emby](https://user-images.githubusercontent.com/26511433/52138123-23dcb380-2601-11e9-832e-acca4ebc79dc.jpg)

**Comment by jianxuanbing** [comment]
current app player can't  Horizontal screen and cpu usage is very much

**Comment by arhamqureshi** [comment]
> Yes, the selection is an option under the user profile's playback option.
> ![screenshot_20190201-090836_emby](https://user-images.githubusercontent.com/26511433/52138123-23dcb380-2601-11e9-832e-acca4ebc79dc.jpg)

Has this option been removed? I can't seem to find it anywhere..

**Comment by anthonylavado** [comment]
That user was describing how it used to work in the Emby app. This option isn’t present in our code, someone needs to add it.

For reference, MxPlayer is built in, but not active (we need these extra options first).

**Comment by arhamqureshi** [comment]
> That user was describing how it used to work in the Emby app. This option isn’t present in our code, someone needs to add it.
> 
> For reference, MxPlayer is built in, but not active (we need these extra options first).

Oh.. my bad. Well this would be great to have.

**Comment by brianjmurrell** [comment]
It's worth noting that the context menu for a video has a _Copy Stream URL_ option which gives you a URL you could use with an external player.  It seems to work on Chrome on Linux but I can't seem to get the URL to paste into a text box on Android in the regular _long-press_->_Paste_ way one usually pastes after one *Copy*/*Cut*s.

Pity, because this would have been a good work-around until this option was implemented.

**Comment by brianjmurrell** [comment]
Actually, I don't think _Copy Stream URL_ is working on Android.  It says it copied the URL however if I already have something in clipboard before I click _Copy Stream URL_ and then _Paste_ what I had in the clipboard before I copied the stream URL pastes.  Sounds like a new ticket I guess.

**Comment by amirasyraf** [comment]
The option 'Enable external video players' exists, but no there's no prompt to select video players when playing a video.

**Comment by dkanada** [comment]
I just pushed a pull request that will disable this button until someone comes along and adds the feature so we can fix all this confusion for now.

**Comment by pras92** [comment]
Yeah, Similar to Emby, Jellyfin has this option as in below pic, but doesn't give a pop-up to choose a preferred external player. I was troubleshooting for sometime, now I know someone added it just in UI, without the functionality. 🤣
![Screenshot_2020-06-17-17-02-16-148_org jellyfin mobile](https://user-images.githubusercontent.com/24977679/84893109-56833100-b0bc-11ea-8cae-27cfee0fd6c0.png)
